src/clr-angular/data/datagrid/datagrid-items.ts

```diff
@@ -12,24 +12,38 @@ import {
     OnChanges,
     SimpleChanges,
     TemplateRef,
-    TrackByFunction
+    TrackByFunction,
+    ViewContainerRef
 } from "@angular/core";
 
 import {Items} from "./providers/items";
 
+export type RowContext<T> = {
+    $implicit: T
```

Comment:
As I mentioned when we were discussing prototypes, we need this to extends `NgForOfContext<T>`. That means you'll have to fill out a few boolean properties and an index when creating the context.

Reply:
Thanks for the `NgForOf` idea. Will address this with that.